src/templates/healthcert/parsers/pdtHealthCertV2/generateMultiQrSection.tsx

```diff
@@ -15,6 +15,7 @@ import {
   isoToDateOnlyString,
   isoToLocaleString,
 } from "../../../../util/datetime";
+import EULogo from "../../eu-dcc-tag-big.png";
```

Comment:
will `eu-dcc-tag-small.png` be used? it looks unused

Reply:
ok will remove it